markdown/setup/dbvisualizer-driver-setup.md

```diff
@@ -31,13 +31,9 @@
 
     c. For the **Driver Type** field, ensure the **Timesteam** is selected.
 
-    d. For the **Database URL** field, enter your [JDBC connection string](../../README.md#optional-connection-properties). For example, ```jdbc:timestream://Region=us-east-1;```
+    d. For the **Database URL** field, enter your [JDBC connection string](../../README.md#optional-connection-properties). For example, ```jdbc:timestream://AccessKeyId=abcdef;SecretAccessKey=123456;Region=us-east-1;```
```

Comment:
Can example string include "SessionToken" as well?

Reply:
The SessionToken is added to the example strings.